packages/preview-middleware-client/src/adp/init-dialogs.ts

```diff
@@ -66,7 +39,7 @@ export const initDialogs = (rta: RuntimeAuthoring, syncViewsIds: string[], minor
  * @returns boolean whether menu item is enabled or not
  */
 export const isControllerExtensionEnabled = (overlays: ElementOverlay[], syncViewsIds: string[], minorUI5Version: number): boolean => {
-    if (overlays.length === 0 || overlays.length > 1) return false;
+    if (overlays.length === 0 || overlays.length > 1) {return false;}
```

Comment:
The formatting looks off here. Can you please run the formatter for the file.

Reply:
Fixed in [0006f42](https://github.com/SAP/open-ux-tools/pull/1941/commits/0006f429474ef300c148b846408dc12ae245623b)